src/windows/volume.cc

```diff
@@ -65,6 +65,41 @@ HRESULT drivelist::volume::GetReadOnlyFlag(const wchar_t letter, BOOL *out) {
   return result;
 }
 
+HRESULT drivelist::volume::HasFileSystem(const wchar_t letter, BOOL *out) {
```

Comment:
The only thing vaguely relevant I could find was https://msdn.microsoft.com/en-us/library/windows/desktop/aa363785(v=vs.85).aspx

I suppose it's conceivable that the only way to check if a volume is properly formatted or not, might be to try mounting it (if it's not already mounted), and then catch any exception that gets thrown? \*shrug\*
Unless you want to use something like https://github.com/resin-io-modules/resin-image-fs but that might be overcomplicating things?